docs/DeveloperGuide.md

```diff
@@ -257,28 +257,46 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
```

Comment:
I don't really understand why this is at the top here? It's important that this message is in our profile but shouldn't it be merged with the story about Marcus? It feels a little out of place here.